packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencedTimestamp.spec.ts

```diff
@@ -59,7 +67,17 @@ describeCompat("GC unreferenced timestamp", "NoCompat", (getTestObjectProvider)
 			this.skip();
 		}
 
-		mainContainer = await provider.makeTestContainer(defaultGCConfig);
+		// We only want to do compat testing for Container Runtime and Data Runtime compat
+		if (!this.test?.parent?.title.includes("runtime")) {
```

Comment:
This is a hard dependency on the format of the test name. Is this a standard?

Reply:
It's true... I don't really think it's a problem though.  There are a couple tests (those that use 1.x Loader) that will fail if they ever started running in error.  The rest are just a waste of time/CPU.